### PR TITLE
Added Mydemo page

### PR DESCRIPTION
**Pull request by vpn1997**
* Added Basic MydemoPage.
* Currently its UI is pretty similar to what we had earlier so I'll work on its layout after demo creation UI.
[ WIP ]

### CODE REVIEW

**Comment by AvaisP** [comment]
There are many overlapping files with the same changes in #225 and this PR. Will cause conflicts once one of them is merged

**Comment by AvaisP** [comment]
@vpn1997 Resolve the conflicts

**Comment by AvaisP** [comment]
@vpn1997 Updates?

**Comment by vpn1997** [comment]
@AvaisP done.Sorry for the delay.

File: Origami/src/components/App.js [hunk 1]
```diff
@@ -55,51 +58,26 @@ class App extends React.Component {
...
+      window.location.pathname.split('/').slice(-1)[0] === 'demo' ||
+      window.location.pathname.split('/')[1] === 'initialsetup'
     ) {
-      this.setState({ displayLogin: "none" });
+      this.setState({ displayLogin: 'none' });
     } else {
-      this.setState({ displayLogin: "" });
+      this.setState({ displayLogin: '' });
     }
-    rootApi
```
Comment (AvaisP) [inline Origami/src/components/App.js]: Why was this removed?

File: Origami/src/components/App.js [hunk 2]
```diff
@@ -120,160 +98,48 @@ class App extends React.Component {
...
     this.clearSessionFlag();
-    window.location = "/";
+    window.location = '/';
   }
 
   getDocs() {
     window.location = ORIGAMI_READ_THE_DOCS;
   }
 
-  handleClickAfterLogin(e, navLinks) {
```
Comment (AvaisP) [inline Origami/src/components/App.js]: This was for the navbar right? What have we replaced it with?

File: Origami/src/components/user/nonghUserProfile.js [hunk 3]
```diff
@@ -219,151 +241,155 @@ class NonGHUserProfileComponent extends React.Component {
...
+                  {this.state.allDeployed.map(row => (
+                    <div key={Math.random()}>
+                      <Row>
+                        {row.map(project => (
+                          <Col span={5} offset={1} key={project.id}>
+                            <Card
+                              style={{ width: '100%' }}
+                              bodyStyle={{ padding: 0 }}
+                            >
+                              <div className="custom-card">
```
Comment (AvaisP) [inline Origami/src/components/user/nonghUserProfile.js]: There isn't a custom card component?

File: Origami/src/components/App.js (hunk 2 above)
Comment (vpn1997) [reply Origami/src/components/App.js]: We are using Default navbar of coreUI. @AvaisP

File: Origami/src/components/user/nonghUserProfile.js (hunk 3 above)
Comment (vpn1997) [reply Origami/src/components/user/nonghUserProfile.js]: Ill add the modified cards in some othe PR.This Pr is just for adding basic mydemo Page.I have raise a PR regarding modifying custom card component for Dashboard(#227 ).We will have similar card for my demo page too.

**Comment by vpn1997** [comment]
image

**Comment by vpn1997** [comment]
@AvaisP can you  please merge this Pr ?
